contents/navigation/navbars/navbar-with-search/index.tsx

```diff
@@ -0,0 +1,180 @@
+import type { FC } from "react"
+import {
+  ActivityIcon,
+  ChevronRightIcon,
+  InfoIcon,
+  Plus,
+  Search,
+  UserIcon,
+} from "@yamada-ui/lucide"
+import {
+  Avatar,
+  Box,
+  Divider,
+  Flex,
+  HStack,
+  IconButton,
+  Indicator,
+  Input,
+  InputGroup,
+  InputLeftElement,
+  InputRightElement,
+  Kbd,
+  LinkBox,
+  LinkOverlay,
+  Spacer,
+  Text,
+  VStack,
+} from "@yamada-ui/react"
+
+const NavbarWithSearch: FC = () => {
+  const tabs = {
+    collections: [
+      { icon: "👍", label: "Sales" },
+      { icon: "🚚", label: "Deliveries" },
+      { icon: "💸", label: "Discounts" },
+      { icon: "💰", label: "Profits" },
+      { icon: "✨", label: "Reports" },
+      { icon: "🛒", label: "Orders" },
+      { icon: "📅", label: "Events" },
+      { icon: "🙈", label: "Debts" },
+      { icon: "💁‍♀️", label: "Customers" },
+    ],
+    general: [
+      { icon: ActivityIcon, label: "Activity", number: "3" },
+      { icon: InfoIcon, label: "Tasks", number: "4" },
+      { icon: UserIcon, label: "Contacts", number: "" },
+    ],
+  }
+
+  return (
+    <VStack
+      as="nav"
+      borderRight="1px solid"
+      borderRightColor="border"
+      fontFamily="-apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, 'Oxygen', 'Ubuntu', 'Cantarell', 'Fira Sans', 'Droid Sans', 'Helvetica Neue', sans-serif"
+      h="full"
+      minH="700px"
+      w="300px"
+    >
+      <LinkBox
+        as="article"
+        cursor="pointer"
+        maxW="sm"
+        transition="all 0.1s"
+        _hover={{ bg: "#F5F5F5" }}
+      >
+        <LinkOverlay>
+          <Flex align="center" gap="md" p="md">
+            <Avatar
+              name="Hirotomo Yamada"
+              src="https://avatars.githubusercontent.com/u/84060430?v=4"
+            />
+            <Flex align="center" gap="lg">
+              <Box>
+                <Text as="b" fontSize="sm">
+                  Hirotomo Yamada
+                </Text>
+                <Text color="gray.400" fontSize="xs">
+                  hirotomo.yamada@avap.co.jp
+                </Text>
+              </Box>
+              <ChevronRightIcon />
+            </Flex>
+          </Flex>
+        </LinkOverlay>
+        <Divider variant="solid" />
+      </LinkBox>
+
+      <VStack as="nav" px="md">
+        <InputGroup size="sm">
+          <InputLeftElement>
+            <Search color="gray.500" />
+          </InputLeftElement>
+          <Input placeholder="Search" pr="4.6rem" />
+          <InputRightElement w="4.6rem">
+            <Text color="#080808">
```

Comment:
Match colors in dark mode as well.
Other text colors in dark mode also refer to Mantine.
```suggestion
            <Text color={["#080808", "white"]}>
```

<img width="294" alt="image" src="https://github.com/user-attachments/assets/c1ba318e-51c6-4127-96ba-1ef76547420a">
<img width="298" alt="image" src="https://github.com/user-attachments/assets/95cdf528-d0f3-4e3e-b652-c2575ba756b2">

Reply:
Opps! Didn't carefully consider that!